README.md

```diff
@@ -34,16 +33,28 @@ Before using ECP with your application/client, you should follow the instruction
 1. Download the ECP binary based on your OS from the latest [Github release](https://github.com/googleapis/enterprise-certificate-proxy/releases).
 
 1. Unzip the downloaded zip and move all the binaries into the following directory:
-   1. Windows: `%AppData%/gcloud/enterprise_cert`.
-   1. Linux/MacOS: `~/.config/gcloud/enterprise_cert`.
+    - Linux/MacOS: `~/.config/gcloud/enterprise_cert`
```

Comment:
This didn't seem to work. Either we need to fix the code to expand the `~` or document that a full path must be used.

FWIW the bundled paths are predictable and installed in the gcloud path.

Reply:
I have updated the configuration file example to use the full path for now.

At the same time, we should fix the code.